sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -167,23 +167,25 @@ export namespace ConnectionContext {
         return Boolean(!this.connection.isOpen() && this.connection.isRemoteOpen());
       },
       async readyToOpenLink() {
-        log.error(`[${this.connectionId}] Waiting until the connection is ready to open link.`);
+        logger.error(`[${this.connectionId}] Waiting until the connection is ready to open link.`);
         // Check that the connection isn't in the process of closing.
         // This can happen when the idle timeout has been reached but
         // the underlying socket is waiting to be destroyed.
         if (this.isConnectionClosing()) {
-          log.error(`[${this.connectionId}] Connection is closing, waiting for disconnected event`);
+          logger.error(
+            `[${this.connectionId}] Connection is closing, waiting for disconnected event`
+          );
           // Wait for the disconnected event that indicates the underlying socket has closed.
           await this.waitForDisconnectedEvent();
         }
 
         // Wait for the connection to be reset.
         await this.waitForConnectionReset();
-        log.error(`[${this.connectionId}] Connection is ready to open link.`);
+        logger.error(`[${this.connectionId}] Connection is ready to open link.`);
```

Comment:
To be clear.. Adding prefixes in the new format is future work?

Reply:
yes, this PR is focusing only on using the new logger and log levels